### ISSUE TITLE
Printable Reports

### ISSUE THREAD

**Issue by IQComputing** [woocommerce/woocommerce#22828]
**Is your feature request related to a problem? Please describe.**
The problem is that reports are not easily printable.

**Describe the solution you'd like**
A more printable report page.

**Describe alternatives you've considered**
Print to PDF

**Additional context**
The WooCommerce reports are a neat way to show data. We can export the data which is nice but the benefit of being able to view the graph would be great. We were able to almost create a more print friendly version of this by scaling the canvas elements but the numbers and dates controlled by flotjs didn't want to cooperate / scale whenever printed.

If a solution with flotjs can't be found then maybe an additional option would be print to PDF instead.

**Comment by kloon** [comment]
cc @woocommerce/isotope

**Comment by woocommercebot** [comment]
Thanks for the suggestion @IQComputing.

We'll keep an eye on the [popularity of this request](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md#feature-requests) :)

**Comment by IQComputing** [comment]
We understand this may be a niche issue but since it was easy/small enough we just decided to make a plugin out of it. [WooCommerce Print Report](https://wordpress.org/plugins/woo-print-report/) is now on the WordPress Repository.
 
Since flotjs isn't printer friendly we ended up using a HTML2Canvas library to capture the output of the graph which seems to work nicely. We're open to suggestions on how we can improve this plugin!
